llvm/test/CodeGen/SPIRV/hlsl-intrinsics/SV_GroupIndex.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: llc -O0 -verify-machineinstrs -mtriple=spirv-vulkan-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-vulkan-unknown %s -o - -filetype=obj | spirv-val %}
```

Comment:
Could you add the target env option to the invocation of spir-val? 
```suggestion
; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-vulkan-unknown %s -o - -filetype=obj | spirv-val --target-env vulkan1.3 %}
```

Or something like that?

Reply:
Sure, done.

Reply:
I'm surprised the tests pass if you are still generating the linkage attribute. If they pass, I will merge.

Reply:
It seem like the validator does not complain about the linkage attribute for Vulkan 1.3, but it does for Vulkan 1.2.